contracts/src/TokenSpoke/TokenSpoke.sol

```diff
@@ -79,41 +73,37 @@ abstract contract TeleporterTokenDestination is
     uint256 public immutable tokenMultiplier;
 
     /**
-     * @notice If {multiplyOnDestination} is true, the source token amount value will be multiplied by {tokenMultiplier} when tokens
-     * are transferred from the source chain into this destination chain, and divided by {tokenMultiplier} when
-     * tokens are transferred from this destination chain back to the source chain. This is intended
-     * when the "decimals" value on the source chain is less than the native EVM denomination of 18.
-     * If {multiplyOnDestination} is false, the source token amount value will be divided by {tokenMultiplier} when tokens
-     * are transferred from the source chain into this destination chain, and multiplied by {tokenMultiplier} when
-     * tokens are transferred from this destination chain back to the source chain.
+     * @notice If {multiplyOnSpoke} is true, the hub token amount will be multiplied by {tokenMultiplier} when tokens
+     * are transferred from the hub into this spoke, and divided by {tokenMultiplier} when tokens are transferred from
+     * this spoke back to its hub.
+     * If {multiplyOnSpoke} is false, the hub token amount value will be divided by {tokenMultiplier} when tokens
+     * are transferred from the hub into this spoke, and multiplied by {tokenMultiplier} when tokens are transferred
+     * from this spoke back to its hub.
      */
-    bool public immutable multiplyOnDestination;
+    bool public immutable multiplyOnSpoke;
 
     /**
-     * @notice Initial reserve imbalance that the token for this destination bridge
-     * starts with. The source bridge contract must collateralize a corresonding amount
-     * of source tokens before tokens can be minted on this contract.
+     * @notice Initial reserve imbalance that the token for this spoke instance starts with. The hub contract must
+     * collateralize a corresonding amount of hub tokens before tokens can be minted on this contract.
      */
     uint256 public immutable initialReserveImbalance;
```

Comment:
DEFINITELY not for this PR 😅 but since we're considering renames, I wonder if this variable wouldn't be better called `initialUndercollateralization`. (That is what it is, right?) There doesn't seem to be any sense of a "reserve" anywhere else besides this variable name, and we're already using the "collateralized" terminology, and "undercollateralization" as a noun does seem to be a legit word :)

Reply:
Yeah that's all correct. But, tbh, I think both accurately enough describe what the variable represents, and that it's well enough documented that the difference between the two relatively pedantic